<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>360</y>
      <width>171</width>
      <height>191</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="btnForward">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>12</y>
       <width>75</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>aNa przód</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnTurnLeft">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>75</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>aW lewo</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnTurnRight">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>75</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>aW prawo</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnGoBack">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>130</y>
       <width>75</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>aW tył</string>
     </property>
    </widget>
   </widget>
   <widget class="QPlainTextEdit" name="txtEditHistory">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>411</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtEditInteracion">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>411</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblNrRoom">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>40</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>a1abc</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="btnNo">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>aNie</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnYes">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>aTak</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>180</y>
      <width>91</width>
      <height>171</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="btnMap">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>75</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>aMapa</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnBackPack">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>75</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>aPlecak</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
